scheduler/kubernetes/kafka-k8s.md

```diff
@@ -0,0 +1,873 @@
+data-kafka-0   Bound     pvc-c405b033-7c4b-11e7-a940-42010a8c0002   3Gi        RWO           portworx-sc-rep2   1m
+data-kafka-1   Bound     pvc-cc70447a-7c4b-11e7-a940-42010a8c0002   3Gi        RWO           portworx-sc-rep2   57s
+data-kafka-2   Bound     pvc-d2388861-7c4b-11e7-a940-42010a8c0002   3Gi        RWO           portworx-sc-rep2   48s
+
+/opt/pwx/bin/pxctl v l
+
+ID                      NAME                                            SIZE    HA      SHARED  ENCRYPTED       IO_PRIORITY     SCALE   STATUS
+523341158152507227      pvc-0e7a636d-7b7b-11e7-a940-42010a8c0002        3 GiB   1       no      no              LOW             0       up - attached on 10.140.0.4
+816480848884203913      pvc-b79e96e9-7b79-11e7-a940-42010a8c0002        3 GiB   1       no      no              LOW             0       up - attached on 10.140.0.5
+262949240358217536      pvc-c405b033-7c4b-11e7-a940-42010a8c0002        3 GiB   2       no      no              LOW             0       up - attached on 10.140.0.3
+733731201475618092      pvc-cc70447a-7c4b-11e7-a940-42010a8c0002        3 GiB   2       no      no              LOW             0       up - attached on 10.140.0.5
+360663112422496357      pvc-d2388861-7c4b-11e7-a940-42010a8c0002        3 GiB   2       no      no              LOW             0       up - attached on 10.140.0.4
+168733173797215691      pvc-faaedef8-7b7a-11e7-a940-42010a8c0002        3 GiB   1       no      no              LOW             0       up - attached on 10.140.0.3
+
+/opt/pwx/bin/pxctl v i pvc-c405b033-7c4b-11e7-a940-42010a8c0002
+
+Volume  :  262949240358217536
+        Name                     :  pvc-c405b033-7c4b-11e7-a940-42010a8c0002
+        Size                     :  3.0 GiB
+        Format                   :  ext4
+        HA                       :  2
+        IO Priority              :  LOW
+        Creation time            :  Aug 8 15:10:51 UTC 2017
+        Shared                   :  no
+        Status                   :  up
+        State                    :  Attached: k8s-2
+        Device Path              :  /dev/pxd/pxd262949240358217536
+        Labels                   :  pvc=data-kafka-0
+        Reads                    :  37
+        Reads MS                 :  8
+        Bytes Read               :  372736
+        Writes                   :  354
+        Writes MS                :  3096
+        Bytes Written            :  53641216
+        IOs in progress          :  0
+        Bytes used               :  65 MiB
+        Replica sets on nodes:
+                Set  0
+                        Node     :  10.140.0.5
+                        Node     :  10.140.0.3
+
+```
+
+Find the Kafka brokers 
+```
+for i in 0 1 2; do kubectl exec -n kafka kafka-$i -- hostname -f; done
+
+kafka-0.broker.kafka.svc.cluster.local
+kafka-1.broker.kafka.svc.cluster.local
+kafka-2.broker.kafka.svc.cluster.local
+```
+
+Create a topic with 3 partitions and which has a replication factor of 3
```

Comment:
Add a heading for this section like "Using Kafka" or "Testing the installation".